pixl_imaging/tests/test_imaging_processing.py

```diff
@@ -112,6 +167,7 @@ async def orthanc_raw(run_containers) -> PIXLRawOrthanc:
 @pytest.mark.processing()
 @pytest.mark.asyncio()
 @pytest.mark.usefixtures("_add_image_to_fake_vna")
+@pytest.mark.usefixtures("_add_image_to_fake_pacs")
```

Comment:
do we need to use this fixture here? and on L189 and L217?

Reply:
oh probably not, well spotted

Reply:
I've removed these unused fixtures too